### DISCUSSION TITLE
Request for template demo for both english and chinese and english documentation

### DISCUSSION THREAD

**Discussion by wail-asad** [note286/xduts#36] [Ideas]
It will be grate if there a demo, this demo will contains  figures, tables, algorithms,chapters , sections , subsections and references.
this demo have corresponding outcome pdf.
also if there english version for documentation.
@note286 you can open repository, then I will create pull request to do all this features.

**Comment by note286** [comment]
I have described it in #19 , and I repeat it here.

---

First of all, I reject all PR. I have described this clearly in the PDF document.

---

Secondly, everyone has their own preferences for dealing with common content such as figure, table, and algorithms. Moreover, I have pointed out the direction for users in the PDF document.

![image](https://user-images.githubusercontent.com/16207292/215296312-56511297-3b82-4a94-ac4f-81df6ccb8289.png)

---

Then, even if I give some examples, they can only be simple, which is meaningless. Users need more detailed explanations and still need to manually view the manuals of corresponding packages.

```latex
\documentclass{xdupgthesis}
\usepackage{graphicx}
\begin{document}
\chapter{title}
\begin{figure}[h]
\centering
\includegraphics[width=.5\linewidth]{example-image}
\caption{title}
\end{figure}
\end{document}
```

![image](https://user-images.githubusercontent.com/16207292/215296393-462a9fd4-a8cf-4666-9b3c-ec81d6b759ea.png)

I think the above example is of no help to users.

---

Next, xduts does not need to provide output PDF.

```latex
\documentclass{xdupgthesis}
\begin{document}
\end{document}
```
Users can generate a empty thesis by above code. This is already the simplest. If the user cannot complete this step, I think he should choose to use Microsoft Word instead of LaTeX to complete his thesis.

---

Finally, Xidian University is a Chinese university. It is a waste of time to develop English documents for a few people who cannot understand Chinese. If you have difficulty reading Chinese, please use [知云文献翻译](http://zhiyunwenxian.cn/) to translate the document.
